app.rb

```diff
@@ -1,7 +1,23 @@
 require 'sinatra/base'
+require './lib/play.rb'
```

Comment:
Opinion: It's normally good practice to leave an empty line after the require statement and before the class definition.